test_external_plugins/snowpark_hello_single_command/pyproject.toml

```diff
@@ -20,7 +20,7 @@ build-backend = "setuptools.build_meta"
 name = "snowflakecli-test-snowpark-hello-plugin"
 requires-python = ">=3.8"
 dependencies = [
-    "snowflake-cli-labs>=1.1.0"
+    "snowflake-cli>=1.1.0"
```

Comment:
Bug reason: CLI was firstly installed in correct dev version but plugins which still required snowflake-cli-labs instead of snowflake-cli were able to find only officially released CLI versions and were overriding dev CLI with official CLI.